velox/dwio/parquet/reader/ParquetReaderUtil.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/dwio/common/compression/Compression.h"
+#include "velox/dwio/parquet/thrift/ParquetThriftTypes.h"
+
+namespace facebook::velox::parquet {
+common::CompressionKind thriftCodecToCompressionKind(
+    const thrift::CompressionCodec::type codec);
+} // namespace facebook::velox::parquet
+
```

Comment:
This is now an extra line. You said format-fix gave you an error?
And no format-fix does not fix/add a new line at the end of the last line of text.
This should just be a new line but now there are new two new line after the end of the comment.

Reply:
Okay , thanks. Is this correct now? 
Because anything else fails the format-check.